tests/modules/humann/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { HUMANN } from '../../../modules/humann/main.nf'
+
+workflow test_humann {
+    
+    input = [
+    	      [ id:'mpa_v30_CHOCOPhlAn_201901' ], // meta map
+              file('https://raw.githubusercontent.com/nf-core/test-datasets/sarek/testdata/umi-dna/qiaseq/SRR7545951-small_1.fastq.gz', checkIfExists: true),
```

Comment:
I would suggest using the `params.test_data['homo_sapiens']['illumina']['test_1_fastq_gz']` here instead of the full link. You can find all test datasets for modules either on https://github.com/nf-core/test-datasets/tree/modules or in the `conf/test_data.config`